src/gbastd/types/rectangle.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RECTANGLE_H
+#define RECTANGLE_H
+
+#include "u.h"
+#include "vector.h"
+
+typedef struct {
+    ivec2 lowerLeftPoint;

```

Comment:
I think is it more reasonable to call it just 'pos'.

Reply:
I understand, 'pos' is much shorter version than 'lowerLeftPoint', but 'pos' sounds a little abstract

Reply:
lowerLeftPoint could be not always true, since rendering/game logic more likely would use rectangle in different ways.(From rendering perspective it would be top left point). So I think is it more reasonable to call it just 'pos'(like position).